lib/lightning/attempts.ex

```diff
@@ -126,6 +126,20 @@ defmodule Lightning.Attempts do
     |> Repo.one()
   end
 
+  @doc """
+  Clears the body and request fields of the dataclip associated with the given attempt.
+  """
+  @spec wipe_dataclip_body(Attempt.t()) :: :ok
+  def wipe_dataclip_body(%Attempt{} = attempt) do
```

Comment:
Perhaps this should be named `wipe_dataclips`, since we're wiping more than the body, and the plural form tells us it's going to do it on potentially more than one given the `update_all`.